outlook/src/taskpane/components/Company/CompanySection/CompanySection.tsx

```diff
@@ -247,6 +255,73 @@ class CompanySection extends React.Component<CompanySectionProps, CompanySection
         );
     }
 
+    private getNoCompanyInsightsSection = () => {
+        let NoInsightsSection = null;
+        const company = this.state.company as Company;
+        if (company.isEmpty()) {
+            if (this.props.partner.isAddedToDatabase() && !this.enriched) {
+                NoInsightsSection = (
+                    <div>v
+                        <div>
+                            {_t("No company attached to this contact")}
+                        </div>
+                        <div className="odoo-secondary-button" style={{margin: "8px auto auto auto"}}
+                             onClick={this.enrichCompanyForPartnerRequest}>
+                            {_t("Create a Company")}
+                        </div>
+                    </div>
+                );
+            } else {
+                if (!this.context.isConnected()) {
+                    NoInsightsSection = (
+                        <div>
+                            {_t("No company linked to this contact could be enriched")}
```

Comment:
Strange to show that if we are not connected at all?

Reply:
Even if the user is not connected, they have access to IAP (for a limited number of requests), which can attempt to enrich the company, in case the enrichment fails (e.g company not found) we display this message.